doc/radosgw/s3/cpp.rst

```diff
@@ -105,7 +105,7 @@ for each bucket.
 		&listServiceCallback
 	};
 	bool header_printed = false;
-	S3_list_service(S3ProtocolHTTP, access_key, secret_key, host, 0, &listServiceHandler, &header_printed);
+	S3_list_service(S3ProtocolHTTP, access_key, secret_key, host, 0, NULL, &listServiceHandler, &header_printed);
```

Comment:
for posterity, 
```c
void S3_list_service(S3Protocol protocol, const char *accessKeyId,
                     const char *secretAccessKey, const char *hostName,
                     S3RequestContext *requestContext,
                     const S3ListServiceHandler *handler,
                     void *callbackData);
```
see https://github.com/ceph/libs3/blob/dcf98ff04bc5dacd5d45854a32870d86dd7b26c7/inc/libs3.h#L1457